.gitlab-ci.yml

```diff
@@ -407,6 +414,7 @@ test_accep_qemux86_64_uefi_grub:
   after_script:
     - cp ${CI_PROJECT_DIR}/../meta-mender/tests/acceptance/results.xml results_accep_qemux86_64_uefi_grub.xml
     - cp ${CI_PROJECT_DIR}/../meta-mender/tests/acceptance/report.html report_accep_qemux86_64_uefi_grub.html
+    - ${CI_PROJECT_DIR}/scripts/maybe-wait-in-stage.sh WAIT_IN_STAGE_TEST ${CI_PROJECT_DIR}/WAIT_IN_STAGE_TEST
```

Comment:
There is a problem here. The `after_script` part will still abort when a command fails. For instance, if the previous step cannot copy the report (maybe it wasn't created), then the wait will never be called.

I don't have a good suggestion on how to do this. Some not-so-good suggestions:
- Pipe all commands in `after_script` to `|| true`
- Just move the wait call to the first thing to do in `after_script` section. This is probably a good idea, as if you are going to manually debug you probably don't care about the artifacts...

Reply:
You're right, it's likely more important that the mechanism works than calling the the very last commands of `after_script`, which you can anyway call manually. I resorted to your second method.